### PR TITLE
Add EIP-6059: Parent-Governed Nestable Non-Fungible Tokens standard

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
This EIP draft proposal includes RMRK team's work on developing an NFT standard where an NFT can own and manage multiple other NFTs.

### CODE REVIEW

File: EIPS/eip-6059.md [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+
+The Parent-Governed Nestable NFT standard extends the [EIP-721](./eip-721.md) by allowing for a new inter-NFT relationship and interaction.
+
+At its core, the idea behind the proposal is simple: the owner of an NFT does not have to be an Externally Owned Account (EOA) or a smart contract, it can also be an NFT.
+
+The process of nesting an NFT into another is functionally identical to sending it to another user. The process of sending a token out of another one involves issuing a transaction from the EOA owning the parent token.
+
+An NFT can be owned by a single other NFT, but can in turn have a number of NFTs that it owns. This proposal establishes the framework for the parent-child relationships of NFTs. A parent token is the one that owns another token. A child token is the token that is owned by another token. A token can be both a parent and child at the same time. Child tokens of a given tokens can be fully managed by the parent token's owner, but can be proposed by anyone.
+
+```mermaid
```
[the remark this answers is not in the document]
Comment (steven2308) [reply EIPS/eip-6059.md:24]: We have our doubts too but let's give a try. If not we'll replace it for another format.

File: EIPS/eip-6059.md [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+
+## Abstract
+
+The Parent-Governed Nestable NFT standard extends the [EIP-721](./eip-721.md) by allowing for a new inter-NFT relationship and interaction.
+
+At its core, the idea behind the proposal is simple: the owner of an NFT does not have to be an Externally Owned Account (EOA) or a smart contract, it can also be an NFT.
+
+The process of nesting an NFT into another is functionally identical to sending it to another user. The process of sending a token out of another one involves issuing a transaction from the EOA owning the parent token.
+
+An NFT can be owned by a single other NFT, but can in turn have a number of NFTs that it owns. This proposal establishes the framework for the parent-child relationships of NFTs. A parent token is the one that owns another token. A child token is the token that is owned by another token. A token can be both a parent and child at the same time. Child tokens of a given tokens can be fully managed by the parent token's owner, but can be proposed by anyone.
```
[the remark this answers is not in the document]
Comment (steven2308) [reply EIPS/eip-6059.md:22]: The idea of the propose-accept pattern is that anyone can send (propose) an NFT to yours, but you're free to accept, reject or ignore it.
Maybe "send" is clearer?

[the remark this answers is not in the document]
Comment (ThunderDeliverer) [reply EIPS/eip-6059.md:22]: I think propose is a good term to be used here, but we can explain it in a bit more detail 🤔

File: EIPS/eip-6059.md [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-11-15
+requires: 165, 721
+---
+
+## Abstract
+
+The Parent-Governed Nestable NFT standard extends the [EIP-721](./eip-721.md) by allowing for a new inter-NFT relationship and interaction.
```
Comment (SamWilsn) [inline EIPS/eip-6059.md:16]:
```suggestion
The Parent-Governed Nestable NFT standard extends [EIP-721](./eip-721.md) by allowing for a new inter-NFT relationship and interaction.
```

File: EIPS/eip-6059.md [hunk 4]
```diff
@@ -0,0 +1,532 @@
...
+
+At its core, the idea behind the proposal is simple: the owner of an NFT does not have to be an Externally Owned Account (EOA) or a smart contract, it can also be an NFT.
+
+The process of nesting an NFT into another is functionally identical to sending it to another user. The process of sending a token out of another one involves issuing a transaction from the account owning the parent token.
+
+An NFT can be owned by a single other NFT, but can in turn have a number of NFTs that it owns. This proposal establishes the framework for the parent-child relationships of NFTs. A parent token is the one that owns another token. A child token is the token that is owned by another token. A token can be both a parent and child at the same time. Child tokens of a given tokens can be fully managed by the parent token's owner, but can be proposed by anyone.
```
Comment (SamWilsn) [inline EIPS/eip-6059.md:22]:
```suggestion
An NFT can be owned by a single other NFT, but can in turn have a number of NFTs that it owns. This proposal establishes the framework for the parent-child relationships of NFTs. A parent token is the one that owns another token. A child token is a token that is owned by another token. A token can be both a parent and child at the same time. Child tokens of a given token can be fully managed by the parent token's owner, but can be proposed by anyone.
```

File: EIPS/eip-6059.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (SamWilsn) [reply EIPS/eip-6059.md:22]: The term is fine, but it isn't defined before this point. As someone whose read EIP-721 and EIP-165 (from the `requires` header), "propose" doesn't really mean anything to me. You should define it in this EIP, or add the EIP with the definition to the `requires` header.

**Review by SamWilsn — COMMENTED** [review]
I think this is almost ready to be merged as a draft. Just need to sort out the two comments.

File: EIPS/eip-6059.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ThunderDeliverer) [reply EIPS/eip-6059.md:24]: Hey @SamWilsn, just a heads up for the future: this didn't render properly, so we had to update it with pngs.

**Pull request merged**